dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyAPI.kt

```diff
@@ -56,12 +57,19 @@ interface CompanyAPI {
         produces = ["application/json"]
     )
     /**
-     * A method to retrieve specific companies identified by their company names
-     * If an empty string is passed as company name, all companies in the data store will be returned.
-     * @param companyName identifier used to search for companies in the data store (can also be an empty string)
+     * A method to retrieve specific companies identified by their company names identifier or stock index
+     * If an empty string is passed as search argument, all companies in the data store will be returned.
```

Comment:
Auch hier nicht lieber "...if ONLY an empty string is passed..." ?

Reply:
geändert